### ISSUE TITLE
Model is always Finetuned in 8bit mode, needs to not be hardcoded 

### ISSUE THREAD

**Issue by danzeeeman** [tloen/alpaca-lora#472]
https://github.com/tloen/alpaca-lora/blob/main/finetune.py#L114

**Comment by tstandley** [comment]
I agree, but I don't think there is a major downside. The lora matricies are stored with much less quantization, so they can fill in the gaps pretty well.
